pom.xml

```diff
@@ -415,6 +415,31 @@ under the License.
             </activation>
         </profile>
 
+        <profile>
```

Comment:
Only a profile about scala_2.13 is OK, not have to bind to `paimon-spark`. Then, to build spark3.x with scala_2.13, use `-Pspark3 -Pscala_213`.